extensions/broker-server/src/test/java/de/sovity/edc/ext/brokerserver/db/TestDatabaseViaEnv.java

```diff
@@ -17,6 +17,10 @@
 import org.apache.commons.lang3.Validate;
 import org.junit.jupiter.api.extension.ExtensionContext;
 
+/**
+ * @deprecated Use test containers wherever possible
+ */
+@Deprecated
```

Comment:
we should either completely purge it, or leave it in (compare the TestDatabaseFactory, where you removed the env stuff.

there might still be some use cases, where you want to use a persistent local DB that you're connected to.

Reply:
ok, deprecation removed